src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
-body {
+@import './blocks/product-card';
+@import './blocks/stars';
+
+* {
+  box-sizing: border-box;
+}
+
+.page_content {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider using SCSS variables for font families to improve maintainability and consistency across your styles. Create a separate file for SCSS variables as required by the task.